test_scripts/API/RAI/050_ATF_N_steeringWheelLocation_IsWrongDataType_RESUME_FAILED_RAI_steeringWheelLocation.lua

```diff
@@ -0,0 +1,279 @@
+	self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession:StartService(7)
+end
+
+commonSteps:RegisterAppInterface("Precondition_for_checking_RESUME_FAILED_RegisterApp")
+commonSteps:ActivationApp(nil, "Precondition_for_checking_RESUME_FAILED_ActivateApp")	
+
+function Test:Precondition_for_checking_RESUME_FAILED_AddResumptionData_AddCommand()
+					
+	local cid = self.mobileSession:SendRPC("AddCommand", { cmdID = 1, menuParams = { position = 0, menuName ="Command 1" },  vrCommands = {"VRCommand 1"} })
+					
+	EXPECT_HMICALL("UI.AddCommand", { cmdID = 1, menuParams = { position = 0, menuName ="Command 1"}})
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+	
+	EXPECT_HMICALL("VR.AddCommand", { cmdID = 1, type = "Command", vrCommands = { "VRCommand 1"} })
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)	
+					
+	EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+	:Do(function()						
+		EXPECT_NOTIFICATION("OnHashChange")
+		:Do(function(_, data)	self.currentHashID = data.payload.hashID end)
+	end)				
+end
+
+function Test:Precondition_for_checking_RESUME_FAILED_CloseConnection()
+	self.mobileConnection:Close() 				
+end
+
+function Test:Precondition_for_checking_RESUME_FAILED_ConnectMobile()
+	os.execute("sleep 30") -- sleep 30s to wait for SDL detects app is disconnected unexpectedly.
+	self:connectMobile()
+end
+
+function Test:Precondition_for_checking_RESUME_FAILED_StartSession()
+	self.mobileSession = mobile_session.MobileSession(
+		self,
+		self.mobileConnection,
+		config.application1.registerAppInterfaceParams)			
+	self.mobileSession:StartService(7)
+end
+
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_RAI_RESUME_FAILED_steeringWheelLocation()
+	config.application1.registerAppInterfaceParams.hashID = "sdfgTYWRTdfhsdfgh"
+	local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_HMICALL("BasicCommunication.ActivateApp", {})
+	:Do(function(_,data) self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {}) end)
+	EXPECT_RESPONSE(CorIdRegister, { success = true, resultCode = "RESUME_FAILED", hmiCapabilities = { steeringWheelLocation = value_steering_wheel_location } })
+
+	EXPECT_NOTIFICATION("OnHMIStatus", 
+		{systemContext="MAIN", hmiLevel="NONE", audioStreamingState="NOT_AUDIBLE"}, 
+		{systemContext="MAIN", hmiLevel="FULL"} )
+	:Times(2)
+	:Timeout(20000)
+
+	EXPECT_HMICALL("UI.AddCommand"):Times(0)
+	EXPECT_HMICALL("VR.AddCommand"):Times(0)
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+function Test.Postcondition_Restore_hmi_capabilities()
```

Comment:
@istoimenova
I propose remove call of function commonPreconditions:RestoreFile("hmi_capabilities.json") taking into account that hmi_capabilities.json was not stored in this test script.